src/main/java/org/recordrobotics/charger/commands/manual/ManualClaw.java

```diff
@@ -0,0 +1,60 @@
+package org.recordrobotics.charger.commands.manual;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.recordrobotics.charger.subsystems.Claw;
+import org.recordrobotics.charger.control.IControlInput;
+
+
+public class ManualClaw extends CommandBase {
+
+	private Claw _claw;
+	private IControlInput _controls;
+
+	private static final double TURN_SPEED = 0.3;
+
+	public ManualClaw(Claw claw, IControlInput controls) {
+		if (claw == null) {
+			throw new IllegalArgumentException("claw is null");
+		}
+		if (controls == null) {
+			throw new IllegalArgumentException("Controls is null");
+		}
+
+		_claw = claw;
+		_controls = controls;
+		addRequirements(claw);
+	}
+
+	@Override
+	public void execute() {
+		switch (_controls.getClawTurn()) {
+			case CUBE:
```

Comment:
why would there be a difference whether we grab cube or cone?  if the difference is the clamping distance, wouldn't it make a difference on the cone where we grab it? not a problem, just curious of the thinking here

Reply:
We just used the same thing as cube until we knew more about how the claw would grab cones